### ISSUE TITLE
aggTrades is WRONG!

### ISSUE THREAD

**Issue by Dylan-Kerler** [jaggedsoft/node-binance-api#436]
```js
const dataOptions = {
       symbol: opts.symbol,
        startTime: startTime,
        endTime: startTime + UTILS.hour
};

const parsedData = await client.aggTrades(dataOptions)
```

gives this error:
```UnhandledPromiseRejectionWarning: Error: apiRequest: Invalid API Key```

Further inspection reveals that the library is the one throwing the error not binance. aggTrades calls marketRequest. And marketRequest throws an error if an api key is not provided:

```js
const marketRequest = ( url, data = {}, callback, method = 'GET' ) => {
        if ( !Binance.options.APIKEY ) throw Error( 'apiRequest: Invalid API Key' );
       // rest of the code snippeted -- //
};
```

But why is this being called? As per the binance documentation aggTrades does not need an api key to be provided so this provides a very annoying error. It is also a breaking change... a lot of my scripts don't work now.

TLDR: aggTrades is not an endpoint that needs an API key. Despite this, the library throws an error if an API key is not provided.

**Comment by jaggedsoft** [comment]
Fixed https://github.com/jaggedsoft/node-binance-api/commit/aac182f038da6baf4266b7a3a6bb8547aec2e9b7

Please note the first parameter is the symbol and the second contains startTime and endTime
```js
    console.info( await binance.aggTrades( 'BTCUSDT', {startTime: new Date().getTime() - 86400, endTime: new Date().getTime() } ) );
```

**Comment by jaggedsoft** [comment]
available in v0.11.5
